soccer/gameplay/evaluation/defensive_positioning.py

```diff
@@ -293,15 +312,15 @@ def goalside_mark_segment(mark_pos, robot, ball=False, kick_eval=None):
 ## Finds the line segment between the mark_pos and the ball
 # @param mark_pos: Point (usually robot position) to defend against
 # @return: LineSegment to defend on
-def ballside_mark_segment(mark_pos, ball_pos=None):
-    #offsets on ball and mark_pos sides
+def ballside_mark_segment(mark_pos: robocup.Point, ball_pos=None):
```

Comment:
return is a `robocup.Line` and `ball_pos` is a `robocup.Point`

Reply:
fixed